src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ReviewDraftHearingRequirementsPreparer.java

```diff
@@ -64,6 +63,8 @@ public boolean canHandle(
 
         decorateWitnessAndInterpreterDetails(asylumCase);
 
+        decorateOutsideEvidenceDefaultsForOldCases(asylumCase);
```

Comment:
This part is done for review hearing requirements, I guess this is new CCD requirements so the fields need to be setup.
Wondering about existing cases when there is Adjustment hearing requirements review. Is that scenario valid?

Reply:
yes, thank you for pointing this, I missed that during review